docs/system/nmw-main-app.md

```diff
@@ -0,0 +1,65 @@
+
+In this way, we ensure that any beacon setup to use the NarrateMyWay standard provides users with useful general information regarding the location they are at.
+
+However, with the setup application, it is also possible for the beacon to provide more enriched information to those who pass by. This is done by utilising the unique UUID of each Bluetooth beacon. This is entirely optional, however should this option be selected during the setup process, this UUID is mapped to more useful information regarding the location and together, this is stored in an external database. The information that may be included here is:
+
+<h4 id="table-2" align="center" class="fs-4 fw-50">
+    <u>Table 2: Optional Additional Information</u>
+</h4>
+
+| Location attribute | Required? | Description                                                                                           |
+| ------------------ | --------- | ----------------------------------------------------------------------------------------------------- | --- |
+| NMW location code  | **Yes**   | The location code according to this standard                                                          |
+| Name               | **Yes**   | A name for the location. This could be, for example the name of a business                            |     |
+| Description        | **Yes**   | A field that must be included to provide the user with more meaningful information about the location |
+| Website            | No        | This is optional and should only be present if the website is accessible for visually impaired users  |
+
+### Extension Packs
+
+As part of our desire to ensure that our application is usable by users without internet connection, any NarrateMyWay beacon detected by the app will at the very least provide users with the information outlined in [Table 1](#table-1) above.
+
+However, with our intended desire to use the NarrateMyWay system within larger organisations, we provide users with the option to download Extension Packs, which allow the more enriched information of [Table 2](#table-2) to be provided to the user.
+
+The intention is that each of these organisations will setup beacons with this additional information, and this is stored externally on a database. Users will then be able to download this information in the form of an Extension Pack, either provided by representatives of the organisation at the location, or downloadable via the application itself as explained below.
+
+### 2. Extension Pack Beacons
+
+As outlined above, it is possible for each beacon setup using NarrateMyWay to provide enriched information which makes use of the UUID of the beacon. If setup to include this additional information, these details along with the UUID is stored in an external database. The purpose of these Extension Pack Beacons, is to allow a user to easily download these packs when they come across such a beacon:
```

Comment:
> *these details along with the UUID is stored in an external database*

=>  "... these details along with the UUID ~is~ **are** stored in an external database"